### ISSUE TITLE
Desktop App Backup does not use selected directory

### ISSUE THREAD

**Issue by MorgothSauron** [standardnotes/desktop#588]
When I enable backup in the desktop app, I select the folder for the backups (`$HOME/Documents/StandardNotes`). 

The problem is that the application will create a sub-directory named `Standard Notes` insides the selected folder. The backups are stored in `$HOME/Documents/StandardNotes/Standard Notes`.

The application should use the folder I selected and not create it's own sub-folder. It's like if Firefox was creating a 'firefox' sub-folder when you select a download folder. 

Standard Notes desktop app should use the folder selected by the user, period. 

As a workaround I edit `backupsLocation` in `$HOME/.config/Standard Notes/user-preferences.json`

**Comment by ackstorm23** [comment]
Such a change would cause backups to store in the root directory unless the user explicitly provided a folder name.

Could this be solved with modified wording during creation making it clear you are picking a parent folder in which to create the Standard Notes folder, and not the destination folder?

**Comment by MorgothSauron** [comment]
The way it works currently is different from what you see in other applications and operating system. I'll try to list few scenarios

1) I select `$HOME/Temp` as the parent folder (I don't go inside the folder)
   - Current behavior: backup directory is `$HOME/Temp/Standard Notes Backups` 
   - Expected behavior: backup directory is `$HOME/Temp`

2) I select `$HOME/Temp` as the backup folder (I go inside the folder)
   - Current behavior: backup directory is `$HOME/Temp/Standard Notes Backups` 
   - Expected behavior: backup directory is `$HOME/Temp`

3) I select `$HOME/Documents/StandardNotes/Backups` as the backup folder (I go inside the folder)
   - Current behavior: backup directory is `$HOME/Documents/StandardNotes/Backups/Standard Notes Backups` 
   - Expected behavior: backup directory is `$HOME/Documents/StandardNotes/Backups`

The fact that SN adds it's own sub-folder is confusing. If I select folder `$HOME/Documents/StandardNotes/Backups` it is expected that the backup will be in that folder, not in a subfolder.

I think I shouldn't have to edit `user-preferences.json` to get the desired path for the backups.

**Comment by JaspalSuri** [comment]
> The fact that SN adds it's own sub-folder is confusing.

I can sympathize with your frustration here. I think it's just to provide a foolproof way for the average user to search for backups of their notes again and avoid any potential conflicts with the OS. @baptiste-grob might be able to share his thoughts on this, since he works on the desktop and web app. :)

**Comment by baptiste-grob** [comment]
Yep what @JaspalSuri is correct. Creating a folder is the safest solution which is why we went with it. But we did consider what you're suggesting. Does manually editing `user-preferences.json` work correctly? If so we can at least be careful to respect your choice there.

**Comment by MorgothSauron** [comment]
Ok. I understand that it works as designed. 

So far I didn't have problem changing the `backupsLocation` directly in `user-preferences.json`. If I open the backup location from the application it opens in the right path. From what I can see the settings will be overwritten only if I use the  "Change Backup Location" menu in the application. Besides this last case, the change made manually in `user-preferences.json` is preserved.
